test/testcafe/framework/page-objects/AuthenticatorExpiredPasswordPageObject.js

```diff
@@ -0,0 +1,13 @@
+import FactorEnrollPasswordPageObject from './FactorEnrollPasswordPageObject';
+
+const requirementsSelector = '.password-authenticator__rules';
+
+export default class AuthenticatorExpiredPasswordPageObject extends FactorEnrollPasswordPageObject {
+  constructor (t) {
+    super(t);
+  }
+
+  getRequirements () {
```

Comment:
shall this method live in `FactorEnrollPasswordPageObject` ?

Reply:
I think it makes sense for it to be in `FactorEnrollPasswordPageObject` so that other views can use it too. Does it still make sense to leave `AuthenticatorExpiredPasswordPageObject` as it without any methods in it for future? Or shall I just switch over to using `FactorEnrollPasswordPageObject` ?

Reply:
use `FactorEnrollPasswordPageObject` and maybe add comment about why it can be reused by other cases like ExpiredPassword.

Reply:
Added comments